.circleci/config.yml

```diff
@@ -13,11 +13,16 @@ jobs:
             login="$(aws ecr get-login --region eu-west-1 --no-include-email)"
             ${login}
       - run:
-          name: Start applications
+          name: Build / pull call centre applications
+          command: |
+            docker-compose pull cla_backend
+            docker-compose build cla_frontend
```

Comment:
🙋‍♂️ If we don't pass an argument in to `docker-compose pull` and `docker-compose build`, it will pull and build everything it needs to, which would lead us to less maintenance (nothing else in the CI configuration mentions applications by name).

Reply:
:thumbsup: